test/test_ngraph_data_cache.cpp

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#include <atomic>
+#include <memory>
+#include "absl/synchronization/barrier.h"
+#include "gtest/gtest.h"
+#include "test/test_utilities.h"
+
+#include "tensorflow/core/common_runtime/optimization_registry.h"
+#include "tensorflow/core/graph/graph_constructor.h"
+#include "tensorflow/core/public/session.h"
+
+#include "ngraph_bridge/ngraph_backend_manager.h"
+#include "ngraph_bridge/ngraph_builder.h"
+#include "ngraph_bridge/ngraph_data_cache.h"
+#include "ngraph_bridge/version.h"
+
+using namespace std;
+namespace ng = ngraph;
+namespace tf = tensorflow;
+
+namespace tensorflow {
+namespace ngraph_bridge {
+namespace testing {
+
+#define ASSERT_OK(x) ASSERT_EQ((x), ::tensorflow::Status::OK());
+#define ASSERT_NOT_OK(x) ASSERT_NE((x), ::tensorflow::Status::OK());
```

Comment:
These can be removed since they are defined in the test_utilities.h
Also, move `#include test/test_utilities.h` from line 20 to line 30 as it is ngraph_bridge file